PatientManagement/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "patientmanagement",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./src/aap.ts"
```

Comment:
Shouldn't this be ".../app.ts" instead of ".../aap.ts"

Reply:
Good catch. I'll fix it tomorrow